tests/test_cli_estimate_area.py

```diff
@@ -0,0 +1,153 @@
+    )
+    assert validated_result.exit_code == 0
+    assert validated_result.output == output
+
+
+def test_cli_estimate_area_valid_features_files_and_1cm_precision():
+    output = '{"km2": "0", "precision": "1cm", "pricing_docs": "For more information, visit https://www.mapbox.com/pricing/#tilesets"}\n'
+    runner = CliRunner()
+    validated_result = runner.invoke(
+        estimate_area,
+        ["tests/fixtures/valid.ldgeojson", "-p", "1cm", "--force-1cm"],
+    )
+    assert validated_result.exit_code == 0
+    assert validated_result.output == output
+
+
+# test scenario for each precision and get expected output. The expected km2 are from Mapbox studio and specifying zooms in the recipe
+def test_cli_estimate_area_valid_fatures_and_10m_precision():
+    output = '{"km2": "1485128", "precision": "10m", "pricing_docs": "For more information, visit https://www.mapbox.com/pricing/#tilesets"}\n'
+    runner = CliRunner()
+    validated_result = runner.invoke(
+        estimate_area, ["tests/fixtures/precisionTesting.ldgeojson", "-p", "10m"]
+    )
+    assert validated_result.exit_code == 0
+    assert validated_result.output == output
+
+
+def test_cli_estimate_area_valid_fatures_and_1m_precision():
+    output = '{"km2": "2562", "precision": "1m", "pricing_docs": "For more information, visit https://www.mapbox.com/pricing/#tilesets"}\n'
+    runner = CliRunner()
+    validated_result = runner.invoke(
+        estimate_area, ["tests/fixtures/precisionTesting.ldgeojson", "-p", "1m"]
+    )
+    assert validated_result.exit_code == 0
+    assert validated_result.output == output
+
+
+def test_cli_estimate_area_valid_fatures_and_30cm_precision():
+    output = '{"km2": "65", "precision": "30cm", "pricing_docs": "For more information, visit https://www.mapbox.com/pricing/#tilesets"}\n'
+    runner = CliRunner()
+    validated_result = runner.invoke(
+        estimate_area, ["tests/fixtures/precisionTesting.ldgeojson", "-p", "30cm"]
+    )
+    assert validated_result.exit_code == 0
+    assert validated_result.output == output
+
+
+def test_cli_estimate_area_valid_fatures_and_1cm_precision():
+    output = '{"km2": "2", "precision": "1cm", "pricing_docs": "For more information, visit https://www.mapbox.com/pricing/#tilesets"}\n'
+    runner = CliRunner()
+    validated_result = runner.invoke(
+        estimate_area,
+        ["tests/fixtures/precisionTesting.ldgeojson", "-p", "1cm", "--force-1cm"],
+    )
+    assert validated_result.exit_code == 0
+    assert validated_result.output == output
+
+
+def test_cli_estimate_area_with_no_validation_flag_and_invalid_feature():
+    runner = CliRunner()
+    # the input is missing "properties"
+    validated_result = runner.invoke(
+        estimate_area,
+        ["--precision", "10m", "--no-validation"],
+        input='{"type": "Feature", "geometry": { "type": "Point","coordinates": [125.6, 10.1]}}',
+    )
+    assert validated_result.exit_code == 0
```

Comment:
why is there no assert on the output here? trying to understand what we return to users when they skip validation and have invalid geojson

Reply:
The validation that's being skipped is that a user's features follows the GeoJSON schema correctly (has type, geometry and properties) but it would throw earlier (line 751) if it was missing features since we extract the features first. But it's missing an output assertion I'll add that in!